it/uk/gov/hmrc/incorporatedentityidentificationfrontend/views/WelshLanguageISpec.scala

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Personally, I'd like a simple test to confirm the Welsh translation is working. Something like

val messagesApi:MessagesApi = app.injector.instanceOf[MessagesApi]

"An example of the welsh text can be retrieved" in {
     messagesApi("service.name.default")(Lang("cy")) mustBe "Gwasanaeth Dilysu Endid"
  }

Probably more for maintenance purposes as much as anything.

Reply:
Added